wire-library/wire-kotlin-generator/src/main/java/com/squareup/wire/kotlin/KotlinGenerator.kt

```diff
@@ -1565,6 +1567,7 @@ class KotlinGenerator private constructor(
         ProtoType.UINT32 to INT,
         ProtoType.UINT64 to LONG,
         ProtoType.ANY to ClassName("com.squareup.wire", "AnyMessage"),
+        ProtoType.DURATION to ClassName("java.time", "Duration"),
```

Comment:
Shouldn't this be the wire package so that MPP works?

Reply:
oops... thank you. There was a few `java.time.Duration` import left as well in a few places.
I cleaned it call.